build/longhaul.vsts-ci.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       sudo -E ./jenkins/longhaul_c.sh
     displayName: 'Build'
   - script: |
-      cd cmake && sudo -E ../build_all/linux/run_tests.sh
+      cd cmake && sudo chmod 755 ../build_all/linux/run_tests_memcheck.sh && sudo -E ../build_all/linux/run_tests_memcheck.sh
```

Comment:
I'm guessing you're doing the chmod because the file says it's not executable. You might have added it to the index without executable permissions? 

https://ionic.zendesk.com/hc/en-us/articles/360000160067-Adding-execute-permissions-using-git#:~:text=Adding%20execute%20permissions%20is%20possible%20on%20any%20OS,after%20running%20git%20add%20and%20before%20git%20commit.

Reply:
The chmod should be able to be removed if you use that ^

Reply:
with executable*